.github/workflows/quickstart_ci.yml

```diff
@@ -43,7 +43,7 @@ jobs:
           mvn -U -B -fae clean install -Drelease
         shell: bash
       - name: Build, run & test ${{ inputs.QUICKSTART_PATH }} Quickstart with provisioned-server profile
-        if: ${{ !inputs.MICROPROFILE }}
+        if: ${{ !fromJSON(inputs.MICROPROFILE) }}
```

Comment:
Leftover from something you were trying?

Reply:
it seems that there is a bug in github action when evaluating boolean